packages/kbn-optimizer/src/optimizer/get_bundles.ts

```diff
@@ -24,7 +24,16 @@ import { Bundle } from '../common';
 import { KibanaPlatformPlugin } from './kibana_platform_plugins';
 
 export function getBundles(plugins: KibanaPlatformPlugin[], repoRoot: string) {
-  return plugins
+  const coreBundle = new Bundle({
```

Comment:
This is hardcoded for now, but in the future we could have a configurable list of entry points to build. I figured no need in building that out until/if we need it. @spalger does that make sense to you?

Reply:
Sort of, except that forcing this function to return a specific bundle at all times is probably part of the reason so many tests are now changed (and some in a way that I don't think is going to work).

Hope this helps, but I've gone ahead an implemented a config option to make this bundle opt-in: https://github.com/joshdover/kibana/pull/30